roles/core/time/molecule/default/verify.yml

```diff
@@ -0,0 +1,19 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  vars:
+    contents: "{{ lookup('file', '/etc/timezone') }}"
```

Comment:
The check can fail depending on the OS: there is no `/etc/timezone` in CentOS 8. The [timezone module](https://docs.ansible.com/ansible/latest/modules/timezone_module.html) uses `timedatectl` to set the time zone.